### PR TITLE
slice part of article fullcontent

### PR DESCRIPTION
**Pull request by khaledkz**
when we get a list of articles if the full content is big it will take a slice of the content

### CODE REVIEW

File: server/public/script/script.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+let articleListContent = document.getElementsByClassName('articleList-content');
+
+    for (var i = 0; i < articleListContent.length; i++) {
+
+      if (articleListContent[i].innerHTML.length > 70) {
```
Comment (kabaros) [inline server/public/script/script.js]: you dont need the `if` check, you can just slice and if it's less then nothing will happen.

**Review by kabaros — CHANGES_REQUESTED** [review]
this is not the right approach. You should never need to use DOM if using react

File: server/public/script/script.js (hunk 1 above)
Comment (khaledkz) [reply server/public/script/script.js]: @kabaros I update it

File: server/public/script/script.js [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (rarmatei) [inline server/public/script/script.js]: I guess we can delete this file, now that we don't need it?

**Pull request merged**